test/performance/shmem_perf_suite/latency_common.h

```diff
@@ -153,7 +153,7 @@ extern void int_element_latency(perf_metrics_t data);
 extern void streaming_latency(int len, perf_metrics_t *data);
 
 void static inline  multi_size_latency(perf_metrics_t data, char *argv[]) {
-    int len;
+    unsigned int len;
     int partner_pe = partner_node(data.my_node);
```

Comment:
Should we make `start_len` and `max_len` in both `latency_common` and `bw_common` `unsigned long int` data type to make it unified?

Reply:
Sure, let's do this after the 1.4.1 release.  Created #669 to track.

Reply:
Ok. thanks. Other than this, the perf suite changes look good to me.